# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../firefox/uname-wrapper.patch
# Copyright (C) 2014 - 2017 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The Python scripts fail with warning output on stderr, ...

	- Rene Rebe <rene@exactcode.de>

--- mozilla-release/media/webrtc/trunk/build/common.gypi.vanilla	2014-08-18 16:54:24.552000097 +0000
+++ mozilla-release/media/webrtc/trunk/build/common.gypi	2014-08-18 16:54:34.984000097 +0000
@@ -117,7 +117,7 @@
             # Anything else gets passed through, which probably won't work very
             # well; such hosts should pass an explicit target_arch to gyp.
             'host_arch%':
-              '<!(uname -m | sed -e "s/i.86/ia32/;s/x86_64/x64/;s/amd64/x64/;s/arm.*/arm/;s/i86pc/ia32/")',
+              '<!(uname -m 2>/dev/null | sed -e "s/i.86/ia32/;s/x86_64/x64/;s/amd64/x64/;s/arm.*/arm/;s/i86pc/ia32/")',
           }],
 
           # Set default value of toolkit_views based on OS.
